docs/subvolume.md

```diff
@@ -0,0 +1,45 @@
+# Subvolume cleanup
+
+The subvolume command is used to clean the stale subvolumes
+which has no parent-pvc attached to it.
+The command would list out all such subvolumes which needs to be removed.
+This would consider all the cases where we can have stale subvolume
+and delete them without impacting other resources and attached volumes.
+
+The subvolume command will require the following sub commands:
+* `ls` : [ls](#ls) lists all the subvolumes
+    * `--type=stale`: lists all the stale volume that can be deleted
+* `delete` : [delete](#delete) stale subvolumes after user's confirmation.
+    * It takes 3 arguments:
+      * list of subvolumes of same filesystem and subvolumegroup
+      * filesystem name
+      * subvolumegroup name
+
+## ls
+
+```bash
+kubectl rook-ceph subvolume ls
+
+# csi-vol-427774b4-340b-11ed-8d66-0242ac110004
+# csi-vol-427774b4-340b-11ed-8d66-0242ac110005
+# csi-vol-427774b4-340b-11ed-8d66-0242ac110006
+# csi-vol-427774b4-340b-11ed-8d66-0242ac110007
+
+```
+
+```bash
+kubectl rook-ceph subvolume ls --type=stale
```

Comment:
How about just the `--stale` flag instead of `--type=stale`?
```suggestion
kubectl rook-ceph subvolume ls --stale
```

Reply:
I tried using it, but just with flags couldn't find the option to add it, it was showing some error, let me know if you can help me, we can discuss this offline.

Reply:
done